# HG changeset patch
# User Ben Abbott  <bpabbott@mac.com>
# Date 1325214099 18000
#      Thu Dec 29 22:01:39 2011 -0500
# Branch stable
# Node ID 4b07db9dd9761296d43a265b5fc575f1ddb859c7
# Parent  2f742be03f908ea029fab3c79ab53a0dfc68e456
accumarray.m: Fix trivial typo in docstring.

diff --git a/scripts/general/accumarray.m b/scripts/general/accumarray.m
--- a/scripts/general/accumarray.m
+++ b/scripts/general/accumarray.m
@@ -102,17 +102,17 @@
 ## can be written with @code{accumarray} as
 ##
 ## @example
 ## accumarray ([@var{i}, @var{j}], @var{sv}', [], [], 0, true)
 ## @end example
 ##
 ## @noindent
 ## For repeated indices, @code{sparse} adds the corresponding value. To
-## take the minimum instead, use @min as an accumulator function:
+## take the minimum instead, use @code{min} as an accumulator function:
 ##
 ## @example
 ## accumarray ([@var{i}, @var{j}], @var{sv}', [], @@min, 0, true)
 ## @end example
 ##
 ## The complexity of accumarray in general for the non-sparse case is
 ## generally O(M+N), where N is the number of subscripts and M is the
 ## maximum subscript (linearized in multi-dimensional case).  If
